src/sagas/activity/sagas.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+
+import uuid from 'uuid4';
+import { call, put, take } from 'redux-saga/effects';
+
+import { doneFetchMessages, messageAdded } from '../../actions/activity';
+import defaultDB from '../../services/database';
+import { convertFromDatabase, convertToDatabase } from '../../utils/mapping/activity';
+import type { ActivityLogMessage } from '../../types/ActivityLogMessage';
+import { createDatabaseUpdateChannel } from '../database';
+
+export function* fetchMessagesSaga(action) {
```

Comment:
We're removing such fetches from database in favor of using realm notification, so that components don't have to know anything about updates and just use redux state. Actually, you did it with `watchNewMessages` saga. To be precise, it's not only watch for new messages, but also updates redux state with initial database state, so it keeps state always up-to-date with database. 

So, `fetchMessagesSaga` can be removed along with actions that handles force update, since we're going to not use that anymore.